tests/test_yaml.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+import pytest
+
+import helper
+from rpmlb.yaml import Yaml
+
+
+@pytest.fixture
+def valid_yaml():
+    """A Yaml object with a valid YAML file."""
+    return Yaml(helper.get_valid_recipe_file())
+
+
+def test_init_loads_file():
+    yaml = Yaml(helper.get_valid_recipe_file())
+    assert yaml
+
+
+def test_init_raises_error_on_not_existing_file():
+    with pytest.raises(FileNotFoundError):
+        Yaml('foo/dummy.yml')
+
+
+def test_dump_prints_output(capsys, valid_yaml):
+    valid_yaml.dump()
+    stdout, stderr = capsys.readouterr()
+    assert 'rh-ror50:' in stdout
+
+
+def test_run_cmds_runs_cmd_by_cmd_element_type_string():
+    # Test with generated file.
+    # Because we can not test using stdout with capsys fixture,
+    # the capsys does not capture stdout from subprocess in run_cmds.
+    random_file = None
+    try:
+        random_file = helper.get_random_generated_tmp_file()
```

Comment:
Can this fail with an exception? If not, i'd suppose you can move it out of the `try` block instead of assigning `None`. The same for the next test.